frontend/javascripts/admin/task/task_create_form_view.js

```diff
@@ -68,46 +70,88 @@ type State = {
 };
 
 export function handleTaskCreationResponse(responses: Array<TaskCreationResponse>) {
+  if (responses.length === 0) {
+    return;
+  }
   const successfulTasks = [];
   const failedTasks = [];
+  let teamName = null;
 
   responses.forEach((response: TaskCreationResponse, i: number) => {
     if (response.status === 200 && response.success) {
-      successfulTasks.push(
-        `${response.success.id},${response.success.creationInfo ||
-          "null"},(${response.success.editPosition.join(",")}) \n`,
-      );
+      if (!teamName) {
+        teamName = response.success.team;
+      }
+      successfulTasks.push(response.success);
     } else if (response.error) {
       failedTasks.push(`Line ${i}: ${response.error} \n`);
     }
   });
-
-  Modal.info({
-    title: `${successfulTasks.length} tasks were successfully created. ${
-      failedTasks.length
-    } tasks failed.`,
-    content: (
-      <div>
-        {successfulTasks.length > 0 ? (
-          <div>
-            Successful Tasks:
-            <pre>
-              taskId,filename,position
-              <br />
-              {successfulTasks}
-            </pre>
-          </div>
-        ) : null}
-        {failedTasks.length > 0 ? (
+  if (successfulTasks.length > 0) {
+    downloadTasksAsCSV(successfulTasks);
+  }
+  if (failedTasks.length > 0) {
+    Modal.info({
+      title: `Failed to create ${failedTasks.length} tasks.`,
+      content: (
+        <div>
           <div>
             Failed Tasks:
             <pre>{failedTasks}</pre>
           </div>
-        ) : null}
-      </div>
-    ),
-    width: 600,
-  });
+        </div>
+      ),
+      width: 600,
+    });
+  }
+}
+
+export function taskToText(task: APITask) {
+  const {
+    id,
+    created,
+    dataSet,
+    type,
+    neededExperience,
+    editPosition,
+    editRotation,
+    status,
+    team,
+    boundingBoxVec6,
+    projectName,
+    script,
+  } = task;
+  const neededExperienceAsString = `${neededExperience.domain},${neededExperience.value}`;
+  const [editPositionAsString, editRotationAsString] = [editPosition, editRotation].map(
+    position => `${position[0]},${position[1]},${position[2]}`,
+  );
+  const totalNumberOfInstances = status.open + status.active + status.finished;
+  const boundingBoxAsString = boundingBoxVec6
+    ? boundingBoxVec6.reduce((partialString, val) => `${partialString}${val},`)
+    : "_,_,_,_,_,_,";
+  const scriptId = script ? script.id : "_";
+
+  const taskAsString =
+    `${id},${created},${dataSet},${type.id},${neededExperienceAsString},${editPositionAsString},` +
+    `${editRotationAsString},${totalNumberOfInstances},${team},${boundingBoxAsString}${projectName},${scriptId}`;
+  return taskAsString;
```

Comment:
Why is this code new/necessary? I guess, you changed the format for the csv? The old format was constructed like this `${response.success.id},${response.success.creationInfo || "null"},(${response.success.editPosition.join(",")}) \n` if I see it correctly (see old line 77). I think, it makes sense to build the csv exactly like the modal content was shown before.

Reply:
The thought I had in mind was that the additional information might be pretty useful to have.

I see two options:
1. Use the format with less information just like it was before.
2. Reorder the new format to have the same order as the previous format but with the additional information appended to it.

What do you think? :thinking: 

For now I'll implement the 1. option

Reply:
I think (1) is the better choice right now, since no one voiced the need for the other ones :)